upup/models/BUILD.bazel

```diff
@@ -12,3 +12,19 @@ go_library(
         "//vendor/github.com/google/cadvisor/pages/static:go_default_library",
     ],
 )
+
+genrule(
+  name = "bindata",
+  srcs = glob(["*","*/*","*/*/*","*/*/*/*","*/*/*/*/*"], exclude=["bindata.go", "vfs.go"]),
```

Comment:
I think you can just do `glob(["**"], exclude=...)` here for the same effect.

Reply:
Thanks